### ISSUE TITLE
Keyboard input is received really quickly

### ISSUE THREAD

**Issue by schas002** [jQueryOS/jQueryOS#1]
**Note**: [this issue was already announced in chat](http://chat.stackexchange.com/transcript/message/28401783#28401783). But we might as well fire it back up here.

After the output test, the next thing is the input test. But the keyboard input is received really quickly. By that I mean really really quickly. By that I mean badly quickly. Uh, whatever.

This was experienced on the [avocados.iso](https://github.com/jQueryOS/AvocadOS/blob/1b36a37d0d67a8649db1da5ac18cb17a798f42b3/avocados.iso) as of commit 1b36a37d in Oracle VM VirtualBox 5.0.0.

**Comment by schas002** [comment]
[Right now it has no delay](http://chat.stackexchange.com/transcript/message/28401786#28401786). That pretty much reveals the cause of the problem. Back to the drawing :boar:, I guess.

**Comment by schas002** [comment]
Also, first issue! \o/

**Comment by schas002** [comment]
Hmm, a possible solution can be to put in the while loop a sleep function.

**Comment by schas002** [comment]
We don't have a sleep function. I'll discuss in chat.

**Comment by somebody1234** [comment]
No, we need to implement interrupts. I'll work on this when I've finished building a self-hosted GCC.

**Comment by schas002** [comment]
I found another possible solution: instead of implementing interrupts from scratch, we can send a typematic byte packet to the keyboard. `send_keyboard(0xf3)` I guess.
